mlir/test/Dialect/SPIRV/IR/primitive-ops.mlir

```diff
@@ -0,0 +1,21 @@
+// RUN: mlir-opt -split-input-file -verify-diagnostics %s | FileCheck %s
```

Comment:
Right now this test doesn't use splits or diagnostics -- we can add those flags later when they become necessary

```suggestion
// RUN: mlir-opt %s | FileCheck %s
```